packages/yew-macro/src/props/component.rs

```diff
@@ -103,20 +56,18 @@ impl ComponentProps {
             None
         };
 
-        let check_props = match self {
-            Self::List(props) => props
-                .iter()
-                .map(|Prop { label, .. }| {
-                    quote_spanned! {label.span()=> __yew_props.#label; }
-                })
-                .collect(),
-            Self::With(with_props) => {
-                let expr = &with_props.expr;
+        let check_props: TokenStream = self
+            .props
+            .iter()
+            .map(|Prop { label, .. }| {
+                quote_spanned! {label.span()=> __yew_props.#label; }
```

Comment:
From the `quote` documentation:

> // On one line, use parentheses.
> let tokens = quote_spanned!(span=> Box::into_raw(Box::new(#init)));

```suggestion
                quote_spanned!(label.span()=> __yew_props.#label;)
```

Reply:
All these suggestions look great to me! Implemented!